src/codebook/data/makeFiltered.js

```diff
@@ -1,12 +1,13 @@
 export function makeFiltered(data, filters) {
   var filtered = data;
   filters.forEach(function(filter_d) {
+    console.log(filter_d);
     //remove the filtered values from the data based on the filters
     filtered = filtered.filter(function(rowData) {
       var currentValues = filter_d.values
         .filter(f => f.selected)
         .map(m => m.value);
-      return currentValues.indexOf(rowData[filter_d.value_col]) > -1;
+      return currentValues.indexOf("" + rowData[filter_d.value_col]) > -1;
```

Comment:
Not sure this fixes, Becca's problem. I think she has values with type = "number" [{id:"1-1-1", age:10},{id:"1-1-2",age:12}] instead of everything  being character [{id:"1-1-1", age:"10"},{id:"1-1-2",age:"12"}] 

Might be wrong, but thinking the change should actually be to `.map(m=>""+m.value)`.